modules/tinymce/src/themes/silver/main/ts/ui/core/complex/BespokeSelect.ts

```diff
@@ -68,6 +68,13 @@ export interface SelectSpec {
   readonly dataset: SelectDataset;
 }
 
+export interface SelectTypeaheadSpec {
+  readonly dataset: SelectDataset;
+  readonly onAction: (item: FormatterFormatItem) => (api: Menu.ToggleMenuItemInstanceApi) => void;
```

Comment:
I'll have a closer look at the actual code, but this `onAction` looks a bit strange.